src/rpc/res.rs

```diff
@@ -31,12 +31,12 @@ enum Content<T> {
 }
 
 impl<T: Serialize> Response<T> {
+	/// Convert and simplify the value into JSON
 	#[inline]
 	fn json(self) -> Json {
```

Comment:
I think we should delete this too...

Reply:
@rushmorem I'm going to leave this separate at the moment, only because the `net::output::simplify()` function is not exported from that module. We can do this in a future PR if necessary. I've changed the function name however, so it has the same name in both modules.